dev/release/publish_homebrew.sh

```diff
@@ -41,7 +50,7 @@ if ! git remote | grep -q --fixed-strings ${github_user}; then
 fi
 
 echo "Updating working copy"
-git fetch --all --prune --tags --force -j$(nproc)
+git fetch --all --prune --tags --force -j$(num_processing_units)
```

Comment:
For Mac, I had to remove the parentheses here, otherwise I got the error:

```
line 55: num_processing_units: command not found
error: switch `j' requires a value
```

```suggestion
git fetch --all --prune --tags --force -j$num_processing_units
```